.bandit

```diff
@@ -0,0 +1,2 @@
+[bandit]
+exclude: /tests,/training
```

Comment:
Any reason to exclude `/training`?

Reply:
There are many security issues with training, fixing it is going to be a project. We should discuss if we need to do this and/or want to do this.

Reply:
OK then. Shall we hold this PR for a couple of days before we make the decision?

Reply:
Training targets WML (or local) and happens in a sandboxed environment - i.e. doesn't expose any public endpoints etc. 

I think we can exclude the `/training` subfolder